templates/example-plans/020-AddressPlan-standard-large-partitioned-queue.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: admin.enmasse.io/v1beta2
+kind: AddressPlan
+metadata:
+  name: standard-large-partitioned-queue
+  labels:
+    app: enmasse
+spec:
+  displayName: Large HA Queue
+  displayOrder: 6
+  shortDescription: Creates a large HA queue backed by a dedicated broker.
+  longDescription: Creates a large HA queue backed by a dedicated broker. The queue is sharded accross multiple brokers for HA and improved performance. A sharded queue no longer guarantees message ordering.
```

Comment:
This "backed by a dedicated broker" isn't true, is it... If the total size is 1.0, but it uses 3 partitions, it will only be using 0.3333 of a broker 3 times

Reply:
Good spotting @rgodfrey . I have updated the text now to 
```  
shortDescription: Creates a large HA queue sharing underlying brokers with other queues.
longDescription: Creates a large HA queue sharing underlying brokers with other queues. The queue is sharded accross multiple brokers for HA and improved performance. A sharded queue no longer guarantees
```